packages/seanime/build.sh

```diff
@@ -20,7 +20,6 @@ termux_step_host_build() {
 }
 
 termux_step_pre_configure() {
-	mkdir $TERMUX_PKG_SRCDIR/web
 	cp -r $TERMUX_PKG_HOSTBUILD_DIR/seanime-web/out $TERMUX_PKG_SRCDIR/web/
```

Comment:
Emmm... I don't think this is a good practice. If you want to copy the folder from `/a/b/c` to `/d/e/f`, it would be better to do the following steps.

```bash
mkdir -p /d/e/f # make sure this folder and its parents exist
cp -R /a/b/c/* /d/e/f/ # copy all the files
```